nvdaHelper/vbufBackends/gecko_ia2/gecko_ia2.cpp

```diff
@@ -281,6 +281,22 @@ bool isLabelVisible(IAccessible2* acc) {
 	return true;
 }
 
+long getChildCount(const bool isAriaHidden, IAccessible2 * const pacc){
+	long rawChildCount = 0;
+	if(!isAriaHidden){
+		auto res = pacc->get_accChildCount(&rawChildCount);
+		if(res != S_OK){
+			rawChildCount = 0;
+		}
+	}
+	return rawChildCount;
+}
+
+bool hasAriaHiddenAttribute(map<wstring,wstring>& IA2AttribsMap){

```

Comment:
Not at all important, but since this function doesn't modify anything, `IA2AttribsMap` may as well be const if that doesn't cause any hoop jumping.